client/main.go

```diff
@@ -388,6 +384,12 @@ func getArkSdkClient(ctx *cli.Context) (arksdk.ArkClient, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	commandName := ctx.Args().First()
```

Comment:
Doing the check here, will then result in concatenating to other err messages ie. `error: error initializing ark sdk client: ` Is this what we wanted? bit ugly

```
ark config
error: error initializing ark sdk client: CLI not initialized, run 'init' cmd to initialize
```

Reply:
maybe we just fmt.Print and return without err?